osquery/tables/networking/darwin/wifi.mm

```diff
@@ -66,6 +68,23 @@ Status getKnownNetworksKey(std::string& key) {
                                                        : INTEGER(0);
   } else if (CFGetTypeID(property) == CFStringGetTypeID()) {
     return stringFromCFString((CFStringRef)property);
+  } else if (CFGetTypeID(property) == CFArrayGetTypeID() && key == "bssids") {
+    std::string results;
+    // Fetch the first BSSID we can find
+    auto count = CFArrayGetCount((CFArrayRef)property);
+    auto cfkey = CFStringCreateWithCString(
+        kCFAllocatorDefault, "LEAKY_AP_BSSID", kCFStringEncodingUTF8);
+
+    for (CFIndex i = 0; i < count; i++) {
+      auto dref =
+          (CFDictionaryRef)CFArrayGetValueAtIndex((CFArrayRef)property, i);
+      CFTypeRef value = nullptr;
+      if (CFDictionaryGetValueIfPresent(dref, cfkey, &value)) {
+        results += stringFromCFString((CFStringRef)value) + ", ";
```

Comment:
Wait, doesn't this leave a trailing ", " at the end? You should prefer to create a set / vector and then `::join` them together with some glue (e.g., ", ").